packages/apps/spaces/app/organizations/[id]/components/Tabs/panels/AccountPanel/AccountPanel.tsx

```diff
@@ -27,7 +27,7 @@ export const AccountPanel = () => {
     <OrganizationPanel title='Account' withFade>
       <RenewalLikelihood
         name={data?.organization?.name || ''}
-        renewalLikelihood={
+        data={
```

Comment:
why that change?

Reply:
Purely subjective choice of being much easier to think about it as server data. Also looks less java-ish :))

Reply:
what about `renewalData` or `renewalLikelihoodData` :D